ekf: {
x0: [0,  # PX
     0,  # PY
     0,  # PZ
     0,  # U
     0,  # V
     0,  # W
     0,  # PHI
     0,  # THETA
     0,  # PSI
     0,  # AX
     0,  # AY
     0,  # AZ
     0,  # BX
     0,  # BY
     0], # BZ
P0: [0.000001,    0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # PX
     0.000001,    0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # PY
     0.1,         0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # PZ
     0.000001,    0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # U
     0.000001,    0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # V
     0.000001,    0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # W
     0.00000001,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # PHI
     0.00000001,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # THETA
     0.00000001,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # PSI
     0.000001,    0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # AX
     0.000001,    0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # AY
     0.000000,    0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # AZ
     0.000001,    0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # BX
     0.000001,    0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # BY
     0.000001],                                 # BZ

Q0: [0.05,           0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # PX
     0.05,           0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # PY
     0.01,           0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # PZ
     0.00020,        0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # U
     0.00020,        0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # V
     0.00020,        0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # W
     0.00025,        0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # PHI
     0.00025,        0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # THETA
     0.00055,        0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # PSI
     0.0000001,    0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # AX
     0.0000001,    0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # AY
     0.0000000,    0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # AZ
     0.00000000051,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # BX
     0.00000000051,  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,  # BY
     0.00000000051],                                 # BZ

R_IMU: [1.536,  0,0,0,
        1.536,  0,0,0,
        1.536],

R_Mocap: [0.01, 0,0,0,0,0,0,
          0.01, 0,0,0,0,0,0,
          0.01, 0,0,0,0,0,0,
          0.01, 0,0,0,0,0,0,
          0.01, 0,0,0,0,0,0,
          0.01],

# R_GPS: [5.1, 0,0,0,0,0,
#         5.1, 0,0,0,0,0,
#         10,  0,0,0,0,0,
#         4,   0,0,0,0,0,
#         .175],

R_GPS: [3.0, 0,0,0,
        3.0, 0,0,0,
        10.0],

inner_loop_rate: 200,
alpha: 0.3
}
